src/content/advanced/configure-dex-in-your-cluster/index.md

```diff
@@ -149,12 +135,16 @@ data:
         connectors:
         - id: customer
           connectorName: test
-          connectorType: microsoft
+          connectorType: github
           connectorConfig: >-
             clientID: <CLIENT-ID-SET-IN-YOUR-IdP>
             clientSecret: <CLIENT-SECRET-SET-IN--YOUR-IdP>
-            tenant: <TENANT-SET-SET-IN--YOUR-IdP>
+            loadAllGroups: false
             redirectURI: https://dex.<CLUSTERID>.<BASEDOMAIN>/callback
+            orgs:
+            - name: main-customer-org
+              teams:
+              - team-infra
```

Comment:
Why this change?

Reply:
Oh, I see. It's supposed to be the GitHub example, but used the wrong `connectorType`.

The rest with `loadAllGroups: false` however I'd still like to understand. Why not load all group memberships?

Reply:
yeah it was a mistake and I have just realized. 

IMO it is a good default, so define which orgs you want to give access instead of pulling all

Reply:
OK for me